Adapter_oM/Settings-Config/AdapterSettings.cs

```diff
@@ -58,6 +58,7 @@ public class AdapterSettings : IObject
 
         [Description("If your adapter does not define DependencyTypes, this can be set to false for performance.")]
         public virtual bool HandleDependencies { get; set; } = true;
+        public virtual bool HandlePriorities { get; set; } = false;
```

Comment:
hmm.. A bit on the fence on this one. Depends on if we say that the priority types should always be tied to also handling dependencies or not. See additional comment below

Reply:
@IsakNaslundBh , thanks for this.
Code will keep HandlePriorities boolean attribute while making sure objects are sorted independently based on dependency first and on priority then.